### ISSUE TITLE
broadcasting with debug

### ISSUE THREAD

**Issue by oneohsixx** [gitnew2018/My-OpenPeriscope#33]
i clicked on debug and notice the option to broadcast and tried it out.. i attempted to stream a video but it came up as a blank broadcast.... is it possible to stream a video with this function and if so what path do i need to paste in? and also, how do we end it?

Any help will be appreciated

**Comment by gitnew2018** [comment]
No, it wasn't working in original openperiscope, I also removed ffmpeg that might have been necessary for it to work.

**Comment by Max104t** [comment]
the script to broadcast is in `bash` I wounder if porting it to `ps` or simple `cmd` may just work?
